docs/source/basics.rst

```diff
@@ -176,6 +176,25 @@ There is more information about creating stubs in the
 The following sections explain the kinds of type annotations you can use
 in your programs and stub files.
 
+Mypy will also look in some default install directory to try to find annotations.
+For example, for UNIX based, with python 3.4, it will try to find it in
+- ``${HOME}/.local/shared/typehints/python3.4/``
+- ``/usr/shared/typehints/python3.4/``
```

Comment:
I'll try to rephrase/intepret what was said, correct me if I'm wrong
 * not having a separated python tree outside of `/usr/lib/python*/site-packages`
 * at least, using UNIX standard install location of installation (not  `/usr/shared` but `/usr/share` for example)

I guess, a way to do it would be to change the PEP to not use `data_files` but rather use `package_data`, as these are closer to the implementation than having a separated tree.

@bdarnell is that okay for you?